### PR TITLE
Add Readme with information about the committee

### PR DESCRIPTION
**Pull request by wordshaker**
## Description of Issue

Add content to the readme so that anyone visiting this repository knows how they can contribute to this group. This includes information about the committee's purpose, how often we meet, how to contribute to the group, who is currently taking part and a link to the code of conduct. 

I've also moved all meeting notes into a meetings folder to help keep the repository tidy.

## What's missing

- Needs information adding on how to get involved in the meetings.
- I'm missing some members names/GitHub links (sorry if I missed you out)

### CODE REVIEW

**Comment by dnfadmin** [comment]
[CLA assistant check](https://cla.dotnetfoundation.org/dotnet-foundation/wg-membership?pullRequest=9) <br/>Thank you for your submission, we really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla.dotnetfoundation.org/dotnet-foundation/wg-membership?pullRequest=9) before we can accept your contribution.<br/><br/>:x: wordshaker [sign now](https://cla.dotnetfoundation.org/dotnet-foundation/wg-membership?pullRequest=9)<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla.dotnetfoundation.org/check/dotnet-foundation/wg-membership?pullRequest=9) it.</sub>

**Comment by wordshaker** [comment]
The ecosystem group has a good readme - should we also add governance and members as sections? https://github.com/dotnet-foundation/ecosystem-growth

File: README.md [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+
+## Contributing
+
+### Raising Ideas
+
+If you have an idea on how the membership offering could be enhanced or any issues you think should be addressed with the current membership offering, please raise an issue on this GitHub Repository
+
+### Attending Membership Meetings
+
+_Info on how to join the committee here._
```
Comment (BillWagner) [inline README.md:27]: Good point @wordshaker 

I think the easiest way to do this is to raise an issue asking to join. I'll watch for those, and add people to meeting invites.

Once this is added, this is ready to merge

**Comment by wordshaker** [comment]
Made the suggested change. Please let me know if it's ok.

**Comment by wordshaker** [comment]
@BillWagner Please can you merge this change

**Comment by BillWagner** [comment]
Thanks for the ping @wordshaker I missed the earlier GitHub notification. Merging now.

**Pull request merged**